### ISSUE TITLE
Support SVG logos for get5 (API already does somewhat)

### ISSUE THREAD

**Issue by tuudik** [splewis/get5#536]
### Expected behavior
PR https://github.com/splewis/get5/pull/498 has made it possible to use SVG logos for team.
get5 should check if SVG logo exists and use it or if it's missing it, download it.

### Actual behavior
get5 works only with png (matchconfig.sp):
```
static void AddTeamLogoToDownloadTable(const char[] logoName) {
  if (StrEqual(logoName, ""))
    return;

  char logoPath[PLATFORM_MAX_PATH + 1];
  Format(logoPath, sizeof(logoPath), "resource/flash/econ/tournaments/teams/%s.png", logoName);
  LogDebug("Adding file %s to download table", logoName);
  AddFileToDownloadsTable(logoPath);
}
```

Sadly I'm very low experienced with SourcePawn, but I guess there should be same check as for API, if `get5_use_svg` is 1, it should check if SVG exists. I dont know how to share CVAR between two plugins. Separate question is then, where should the CVAR be located, which config.

- Plugin version: 0.7.2-dev 8890db8
- Sourcemod version: 1.10

**Comment by tuudik** [comment]
@MoritzLoewenstein Perhaps you have time to check this issue too?

**Comment by MoritzLoewenstein** [comment]
I will check it out as soon as I have time

**Comment by splewis** [comment]
Should be  resolved by #538 now.

**Comment by MoritzLoewenstein** [comment]
Unless I am missing something this does not get solved via #538, because the legacy logo path is hardcoded in the `Format(...)`, but the svg are under the normal path.
